event-processing/event-processing-template.yml

```diff
@@ -229,9 +229,9 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       FunctionName: "EventProcessorFunction-KBV"
-      CodeUri: ../event-processing/event-processor
+      CodeUri: ../event-processing/lambda
       PackageType: Zip
-      Handler: app.handler
+      Handler: app.eventProcessorHandler
```

Comment:
Small point, but I would rename the classes and keep the method name the same. Eg.
```
eventProcessorApp.handler
obfuscatorApp.handler 
```
etc

Reply:
So at the moment they are all in the same file. But yes I can look into having a separate class per handler if it is preferred

Reply:
@des-hippo Take a look at the changes I've made for this.